portal/portal-impl/impl/src/bundle/sitenav.properties

```diff
@@ -115,13 +115,13 @@ pda_back_to_sites = Back to sites list
 pda_subsite_lead = --  
 pda_subsite = (Sub Site)
 
-join_title = Joinable Site
-join_instr = You are not currently a member of the site:
-join_question = Would you like to join the site?
+join_title = Join this Site
```

Comment:
"Join this Site" should be all capitals ("Join This Site") to match the xlogin page's title.